t/rose-app-run/28-git.t

```diff
@@ -241,7 +241,12 @@ source=git:$TEST_DIR/zz9+zα/::earth/::v1
 __CONFIG__
 run_fail "$TEST_KEY" rose app-run --config=../config
 file_cmp "$TEST_KEY.err" "$TEST_KEY.err" <<__ERROR__
-[FAIL] file:hello/fruit_main=source=git:$TEST_DIR/zz9+zα/::earth/::v1: ls-remote: could not locate '$TEST_DIR/zz9+zα/'
+[FAIL] file:hello/fruit_main=source=git:$TEST_DIR/zz9+zα/::earth/::v1: ls-remote: could not locate '$TEST_DIR/zz9+zα/':
```

Comment:
I'm mildly concerned that this is vulnerable to Git error messages changing. Would it be better as a `grep_ok`?

Reply:
I'm surprised this change didn't break this test?